libmscore/excerpt.cpp

```diff
@@ -1142,13 +1142,49 @@ QList<Excerpt*> Excerpt::createAllExcerpt(MasterScore *score)
                   for (int i = part->startTrack(), j = 0; i < part->endTrack(); i++, j++) {
                         e->tracks().insert(i, j);
                         }
-                  QString name = createName(part->partName(), all);
+                  QString name = part->partName();
```

Comment:
`Excerpt::createAllExcerpt()` and `ExcerptsDialog::createName()` are the only callers of `Excerpt::createName()`, both should benefit from the automatic numbering I think?